access/src/main/webapp/WEB-INF/jsp/fullRecord/neighborList.jsp

```diff
@@ -30,7 +30,7 @@
 				<c:if test="${briefObject.id == neighbor.id}"> current_item</c:if>
 			</c:set>
 			<div class="relateditem ${currentItemClass}">
-				<div class="relatedthumb">
+				<div class="relatedthumb ${isDeleted} ${currentItemClass}">
```

Comment:
Why is `currentItemClass` being added in here now? It doesn't look like anything has changed with the class. Was there a bug?

Reply:
Nope, got copy/pasted